packages/expo-updates/ios/EXUpdates/EXUpdatesDevLauncherController.m

```diff
@@ -0,0 +1,119 @@
+//  Copyright © 2021 650 Industries. All rights reserved.
+
+#import <EXUpdates/EXUpdatesAppController+Internal.h>
+#import <EXUpdates/EXUpdatesAppLauncherWithDatabase.h>
+#import <EXUpdates/EXUpdatesConfig.h>
+#import <EXUpdates/EXUpdatesDevLauncherController.h>
+#import <EXUpdates/EXUpdatesReaper.h>
+#import <EXUpdates/EXUpdatesReaperSelectionPolicyDevelopmentClient.h>
+#import <EXUpdates/EXUpdatesRemoteAppLoader.h>
+#import <EXUpdates/EXUpdatesSelectionPolicy.h>
+#import <EXUpdates/EXUpdatesUpdate.h>
+#import <React/RCTBridge.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+static NSString * const EXUpdatesDevLauncherControllerErrorDomain = @"EXUpdatesDevLauncherController";
+
+@implementation EXUpdatesDevLauncherController
+
+@synthesize bridge = _bridge;
+
++ (instancetype)sharedInstance
+{
+  static EXUpdatesDevLauncherController *theController;
+  static dispatch_once_t once;
+  dispatch_once(&once, ^{
+    if (!theController) {
+      theController = [EXUpdatesDevLauncherController new];
+    }
+  });
+  return theController;
+}
+
+- (void)setBridge:(nullable id)bridge
+{
+  _bridge = bridge;
+  if ([bridge isKindOfClass:[RCTBridge class]]) {
+    EXUpdatesAppController.sharedInstance.bridge = (RCTBridge *)bridge;
+  }
```

Comment:
Is there a valid `bridge` that is not also an RCTBridge?


Shouldn't both of these 'set' operations use this check?

Reply:
The `bridge` property on this class is actually of type `id` rather than `RCTBridge`, so that `expo-updates-interface` doesn't need to have a dependency on the React pod. Given that, this behavior (accepting any object on this class, and only passing it along to EXUpdatesAppController if it has the type that class expects) made the most sense to me.